charts/sda-db/values.yaml

```diff
@@ -25,8 +24,8 @@ externalPkiService:
 extraSecurityContext: {}
 
 image:
-  repository: ghcr.io/neicnordic/sda-db
-  tag: v2.1.10
+  repository: ghcr.io/neicnordic/sensitive-data-archive
+  tag: v0.0.44-postgres
```

Comment:
I'm curious how do people easily figure out which version to use after some changes been made. It seems the tag is automatically bumped and there are many of them.

Reply:
A helm release uses a specific set of containers, so unless you are doing testing or need the absolute latest images you don't need to worry.